src/main/java/seedu/address/model/person/SortOrder.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.person;
+
+/**
+ * Represents the sort order of the client list.
+ */
+public enum SortOrder {
```

Comment:
I like that you abstracted out SortCriteria and SortOrder into their own classes!